edith-step/src/ReceiptInput.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+
+export default class ReceiptInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {items: [], text: '', price: 0.0};
```

Comment:
can we change the name 'text' to something more specific ? 
is it the receipt name ?

Reply:
it's the name of the grocery item, i'll change that